File: src/main/java/com/gitlab/daring/image/command/feature/FlannMatcherCommand.java
package com.gitlab.daring.image.command.feature;

import com.gitlab.daring.image.command.BaseCommand;
import com.gitlab.daring.image.command.CommandEnv;
import org.bytedeco.javacpp.opencv_features2d.FlannBasedMatcher;

public class FlannMatcherCommand extends BaseCommand {

    //TODO support all params

    public FlannMatcherCommand(String... args) {
        super(args);
    }

    @Override
    public void execute(CommandEnv env) {
        env.descriptorMatcher = new FlannBasedMatcher();
    }

}

File: src/main/kotlin/com/gitlab/daring/image/command/feature/FlannMatcherCommand.kt
package com.gitlab.daring.image.command.feature

import com.gitlab.daring.image.command.CommandEnv
import com.gitlab.daring.image.command.KBaseCommand
import org.bytedeco.javacpp.opencv_features2d.FlannBasedMatcher

internal class FlannMatcherCommand (vararg args: String) : KBaseCommand(*args) {

    //TODO support all params

    override fun execute(env: CommandEnv) {
        env.descriptorMatcher = FlannBasedMatcher()
    }

}